respec/organisation-config.mjs

```diff
@@ -645,6 +645,24 @@ export function loadRespecWithConfiguration(localConfig) {
     ...localConfig.localBiblio,
   };
 
+  respecConfig.preProcess.push((config, document, utils) => {
+    if (!config.alternateFormats) {
+      config.alternateFormats = [];
+    }
+    const pdfName = `${config.pubDomain}-${config.shortName}-${config.publishVersion}.pdf`;
+    const existingFormat = config.alternateFormats.find(format => format.label === 'pdf');
+    if (existingFormat) {
+      if (existingFormat !== pdfName) {
+        utils.showError(`Invalid name for PDF format. Expected "${pdfName}", but got "${existingFormat}"`);
+      }
+      return;
+    }
+    config.alternateFormats.push({
```

Comment:
Is het ook getest wanneer de `alternateFormats` array nog in de local config staat (en dus hiermee dubbel)?